### PR TITLE
API-14969 Added patient Frank L. Arnold and associated records.

### PR DESCRIPTION
**Pull request by taylorrassmannlibertyits**
https://vajira.max.gov/browse/API-14969

NOTE: I kept the files in pretty json format for review. I will squish them before final merge.

### CODE REVIEW

File: datamart/dm-records-1012667179V787205/dmMedOrd1012667179787205000O.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  "objectType": "MedicationOrder",
+  "objectVersion": "1",
+  "cdwId": "1012667179787205000:O",
+  "patient": {
+    "type": "Patient",
+    "reference": "1012667179V787205",
+    "display": "Frank L. Arnold"
+  },
+  "dateWritten": "2021-07-06",
+  "status": "COMPLETED",
```
Comment (jblefkove-va) [inline datamart/dm-records-1012667179V787205/dmMedOrd1012667179787205000O.json:11]: Looking at the [transformer mapping](https://github.com/department-of-veterans-affairs/health-apis-data-query/blob/master/data-query/src/main/java/gov/va/api/health/dataquery/service/controller/medicationrequest/R4MedicationRequestFromMedicationOrderTransformer.java#L44), I don't think this is correct?

Comment (taylorrassmannlibertyits) [reply datamart/dm-records-1012667179V787205/dmMedOrd1012667179787205000O.json:11]: dang so I was right that "COMPLETED" wasn't supported! I should have looked at the right string though. Thanks!

File: datamart/dm-records-1012667179V787205/dmMedOrd1012667179787205004O.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  "dosageInstruction": [
+    {
+      "dosageText": "TAKE ONE TABLET BY MOUTH AT BEDTIME AS NEEDED FOR SLEEP",
+      "timingText": "As directed by physician.",
+      "routeText": "ORAL"
+    }
+  ],
+  "dispenseRequest": {
+    "numberOfRepeatsAllowed": 2,
+    "supplyDurationUnits": "days"
```
Comment (jblefkove-va) [inline datamart/dm-records-1012667179V787205/dmMedOrd1012667179787205004O.json:26]: Is `supplyDurationUnits` necessary here?

**Comment by jblefkove-va** [comment]
This one's ready to minify

**Review by jblefkove-va — APPROVED** [review]

**Pull request merged**